q2_fondue/types/_format.py

```diff
@@ -100,11 +100,27 @@ def _validate_id(self, _id: str):
 
     def _validate_(self, level):
         df = pd.read_csv(str(self), sep='\t')
+        cols_df = df.columns.tolist()
 
-        if df.shape[1] > 1:
+        if df.shape[1] > 2:
+            print(df.columns.tolist())
             raise ValidationError(
                 'NCBI Accession IDs artifact should only contain a single '
-                'column with IDs of the SRA runs or NCBI\'s BioProjects.'
+                'column with IDs of the SRA runs or NCBI\'s BioProjects and '
+                'an optional column with associated DOIs.'
+            )
+        elif df.shape[1] == 2 and (
+                'DOI' not in cols_df and 'doi' not in cols_df):
+            raise ValidationError(
+                'NCBI Accession IDs artifact with two columns '
+                'must contain the column \'DOI\'.'
+            )
```

Comment:
I think I'd just put it under a single `if` statement branch (together with the condition above) and raise a single error (the one above, maybe just stating explicitly the name of the doi column). Or keep it separate but just show the same message - it's not like the second one is _that_ much different in the end...